openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -446,11 +446,31 @@ def process(self, instance):
         self.scene_path = file_path
         self.log.info("Using {} for render/export.".format(file_path))
 
-        self.job_info = self.get_job_info()
+        # Check whether we are splitting render job in export + render
+        # and if so, create a separate task for the render
+        # TODO: Find a cleaner way so `get_job_info` can take extra args
+        # depending on host
+        export_job = instance.data["exportJob"]
+        self.job_info = self.get_job_info(
+            split_render_job=export_job, export_job=True
+        )
```

Comment:
⚠️ This is a massive TODO actually. Changing the signature of this method means all inherited submission plug-ins will need to follow along. This change will currently break all non-houdini submissions using the AbstractSubmitDeadline class.

Reply:
yeah certainly, I created the PR to discuss what other ways you guys envision implementing this in a host agnostic way. I think that adding `dependency_job_ids` to all submission plug-ins would be quite alright as that's a core feature of job info deadline payload and perhaps we could also expose a `plugin` arg and `**kwargs` to pass some optional keys?
However... I wouldn't want to move this logic:
```python
plugin = instance.data["family"].replace("_rop", "").capitalize()
```
To the `abstract_submit_deadline` as that's specific to Houdini

Reply:
I have moved some of the original logic to the respective Houdini functions so the function interfaces could stay much more agnostic. As I said, this same functionality of splitting export + render tasks should be implemented in most other hosts so it's not like it's something that will stay specific to just Houdini, separating a render job in two parts is a very common workflow